taiga_halo2/src/transaction.rs

```diff
@@ -42,6 +42,7 @@ pub struct ShieldedPartialTxBundle(Vec<ShieldedPartialTransaction>);
 #[derive(Debug, Clone, PartialEq, Eq)]
 #[cfg_attr(feature = "nif", derive(NifStruct))]
 #[cfg_attr(feature = "nif", module = "Taiga.Transaction.Result")]
+#[cfg_attr(feature = "borsh", derive(BorshSerialize, BorshDeserialize))]
```

Comment:
Probably also should have
`#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]`